src/realtime/realtime-hapi-plugin.ts

```diff
@@ -119,6 +119,7 @@ export class RealtimeHapiPlugin {
       },
       config: {
         bind: this,
+        auth: 'jwt-url',
```

Comment:
The constructor of this file could also be improved (as were other files). The imports are also a bit weirdly formatted, along with an empty line at the top of the file.